### DISCUSSION TITLE
How do MotionCor3 and AreTomo2 integrate with nextpyp?

### DISCUSSION THREAD

**Discussion by jpellman** [nextpyp/.github#62] [General]
Greetings nextpyp developers,

One of the researchers at my center recently requested that I add MotionCor3/AreTomo2 to our SLURM cluster so that they can run alongside nextpyp 0.6.0, but I couldn't find anything in the documentation about how these tools are hooked into / integrated with nextpyp.  Are they built into the Singularity image, or do they need to be bind-mounted in somehow?

Many thanks,
John Pellman

**Comment by jpellman** [comment]
Apologies for the noise- I just noticed that the Singularity image definition file was updated last week to include these two programs (https://github.com/nextpyp/pyp/blame/551c0923d1a9ed5054aa7604355efa9bbd2c91cd/singularity/pyp.def#L19).

**Reply by abartesaghi** [reply]
Indeed, everything ships inside the container, so there is no need to install the programs separately.
